src/App.js

```diff
@@ -40,16 +40,20 @@ class App extends Component {
   }
 
   onItemClicked = () => {
-    // Fill this in for Wave 3!
+    let addPoint = this.state.points + 1;
+    this.setState({
+      points: addPoint,
+    })
   }
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
+              type = {item.type} // HJ Addition
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
+               add={this.onItemClicked}
```

Comment:
Make sure that you are highlighting what the purpose of that function is with its name. If it’s a callback, use the term callback, if it’s an event handler, tell us what kind of event it is.

Suggestion: Save `onItemClicked` for the event handler in `GameItem.js` and add `Callback` to the `add` name.